.github/workflows/wheels.yaml

```diff
@@ -79,7 +79,12 @@ jobs:
     name: Publish to PyPI
     runs-on: ubuntu-latest
     needs: build_wheels
-    if: startsWith(github.ref, 'refs/tags/')
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
+    environment:
+      name: pypi
+      url: https://pypi.org/p/ompl
+    permissions:
+      id-token: write
```

Comment:
nit: does this do anything within this PR, or is it just setting up for future work?

Reply:
It's set-up for the next release. When we tag 1.7.1, I'd like to get the auto-push to PyPI working.